lib/betagouvbot/format_mail.rb

```diff
@@ -15,7 +15,7 @@ def initialize(subject, body_t, recipients, sender)
     # @note Email data files consist of 1 subject line plus body
     def self.from_file(body_path, recipients = [], sender = 'secretariat@beta.gouv.fr')
       subject, *rest = File.readlines(body_path)
-      new(subject.strip, rest.join, recipients, sender)
+      new(subject.to_s.strip, rest.join, recipients, sender)
     end
 
     def call(context)
```

Comment:
Or, alternatively, make them public and change class name and remove `.call`:

```ruby
class MailFormatter
  module_method

  class << self
    def format_recipients(context)
       # ...
    end

    def format_message(context)
      # ...
    end

    private

    # ...
  end
end
```

WDYT?